application/actions/auth_test.go

```diff
@@ -28,22 +28,22 @@ func (as *ActionSuite) Test_AuthLogin_Invite() {
 	}{
 		{
 			name: "Bad Invite Code",
-			queryParams: fmt.Sprintf("%s=123456&%s=badInviteCode",
-				ClientIDParam, InviteCodeParam),
+			queryParams: fmt.Sprintf("%s=badInviteCode",
+				InviteCodeParam),
```

Comment:
This and other lines can be unwrapped
```suggestion
			queryParams: fmt.Sprintf("%s=badInviteCode", InviteCodeParam),
```